bootstrap/update-if-crawler.sh

```diff
@@ -17,4 +16,6 @@ fi
 
 echo "*	Creating new container $SERVICE_NAME"
 
-docker run -p $PORT_EXTERNAL:$PORT_INTERNAL -d --name $SERVICE_NAME docker-registry.sitesearch.cloud/$DOCKER_IMAGE > nul
+docker run --name $SERVICE_NAME \
+	-d --network $NETWORK_NAME  \
+	docker-registry.sitesearch.cloud/$DOCKER_IMAGE > /dev/null
```

Comment:
With "-d" you do not have any standard output when running docker containers. So  `> /dev/null` might be superfluous here.